src/fsharp/fsc.fs

```diff
@@ -710,8 +710,7 @@ module AttributeHelpers =
              if deterministic && versionString.Contains("*") then
                  errorR(Error(FSComp.SR.fscAssemblyWildcardAndDeterminism(attribName, versionString), Range.rangeStartup))
              try Some (IL.parseILVersion versionString)
-             with e -> 
-                 warning(Error(FSComp.SR.fscBadAssemblyVersion(attribName, versionString), Range.rangeStartup))
+             with e ->
```

Comment:
Please add a comment that the warning will be reported by the TypeChecker.fs

Reply:
done.